data/json/items/melee/swords_and_blades.json

```
@@ -1215,7 +1216,7 @@
     "longest_side": "80 cm",
     "bashing": 2,
     "cutting": 25,
-    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "neutral" },
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "good" },
```

```suggestion
    "to_hit": { "grip": "solid", "length": "short", "surface": "line", "balance": "good" },
```
hidden weapon means not a perfect grip

---

I would say the cane and the hidden sword should have the same grip. I was thinking a cane with the 90* head and spot for a finger to wrap around the front since that's mostly what I saw when I looked up sword canes.

---

I don't have any hard evidence to back this up other than sword and stage fighting with between the SCA and renfaires for heavy rapier fighting and having held sword canes and it feels like the grip you describe would be easier to get knocked from your hand than what I would consider a weapon grade grip.  Does that make sense?  Or are you saying that I should have hit both with the downgrade because if I missed one you are absolutely correct.

---

I was saying that it should be weapons grade but if you have actual experience with them then I'll trust your word on it and change both.